sdk/cosmos/azure-cosmos/azure/cosmos/_retry_utility.py

```diff
@@ -36,10 +37,11 @@
 from . import _session_retry_policy
 from . import _gone_retry_policy
 from . import _timeout_failover_retry_policy
+from . import _container_recreate_retry_policy
 from .http_constants import HttpHeaders, StatusCodes, SubStatusCodes
 
 
-# pylint: disable=protected-access
+# pylint: disable=protected-access, disable=too-many-lines, disable=too-many-statements, disable=too-many-branches
```

Comment:
Isn't linter just a suggestion - like in IntelliJ it is just a yellow squiggly line, doesn't affect compilation - does it affect interpretation here?

Reply:
It's part of the Checks for code standards in python projects. This was just to match it with the async version as it's sometimes unavoidable to exceed the recommended lines of codes for some files.